cmd/restic/cmd_backup.go

```diff
@@ -98,6 +100,8 @@ func init() {
 	f.StringVar(&backupOptions.FilesFrom, "files-from", "", "read the files to backup from file (can be combined with file args)")
 	f.StringVar(&backupOptions.TimeStamp, "time", "", "time of the backup (ex. '2012-11-01 22:08:41') (default: now)")
 	f.BoolVar(&backupOptions.WithAtime, "with-atime", false, "store the atime for all files and directories")
+	f.StringVar(&backupOptions.RootPrefix, "prefix", "", "apply a prefix to target paths")
+	f.IntVar(&backupOptions.RootStrip, "strip-components", 0, "strip NUMBER leading components from target paths")
```

Comment:
Use `n` instead of `NUMBER` and surround it with backticks, then it'll be printed correctly in the help:
```go
f.IntVar(&backupOptions.RootStrip, "strip-components", 0, "strip `n` leading components from target paths")
```

Reply:
Okay.